src/Forms/AssessmentSettingsForm.cs

```diff
@@ -270,7 +269,9 @@ public KeyValuePair<string, string> GetAssessmentDuration()
         public KeyValuePair<string, string> GetSelectedOptimizationPreference()
         {
             KeyValuePair<string, string> empty = new KeyValuePair<string, string>("", "");
-            if (OptimizationPreferenceComboBox.SelectedItem == null)
+            if (mainFormObj.IsAvsBusinessProposalSelected())
+                return new KeyValuePair<string, string>("MigrateToAvs", "AVSOnly");

```

Comment:
Why is this hard-coded.

The idea here is that only a null check needs to be there and then the selected optimization preference in the box can be returned.

Reply:
It is hard- coded because in case the user selects Quick AVS Proposal, the migration strategy becomes disabled. If I was passing it as empty strings, the validation was failing at multiple places, and it was also making the preference a bit ambiguous. Hence, to remove the ambiguity and making a more explicit choice for creating assessments and business case settings, I hard-coded the value.

In future, if we incorporate SAP, it would be easy for us to make the changes now.

Reply:
Will discuss async on call.